WcaOnRails/app/views/competitions/_nav.html.erb

```diff
@@ -174,7 +174,7 @@
                   <% nav_item[:tiny_children].each do |child_nav_item| %>
                     <%= link_to child_nav_item[:path],
                                 title: child_nav_item[:title],
-                                class: (child_nav_item[:active] ? " active" : ""),
+                                class: "nav_event" + (child_nav_item[:active] ? " active" : ""),
```

Comment:
uber nit which I'm not going to let keep me from merging this up: We try to use dashes instead of underscores in css class names, element ids, and urls.

Reply:
fixing!

Reply:
Done.